admin_guide/cluster_management.rst

```diff
@@ -307,14 +336,6 @@ To require that all connections supply a hashed password, update the PostgreSQL
   host    all             all             127.0.0.1/32            md5
   host    all             all             ::1/128                 md5
 
-The coordinator node needs to know roles' passwords in order to communicate with the workers. Add a `.pgpass <https://www.postgresql.org/docs/current/static/libpq-pgpass.html>`_ file to the postgres user's home directory, with a line for each combination of worker address and role:
```

Comment:
Community users might still want to follow these instructions given that they can't use `pg_dist_authinfo`.